test/e2e/main_test.go

```diff
@@ -27,6 +27,8 @@ var (
 	numberOfBYOHNodes int
 	// privateKeyPath is the path of the private key file used to configure the Windows node
 	privateKeyPath string
+	// inTreeUpgrade indicates if tests should be ran to test upgrading from an in-tree storage solution to CSI
```

Comment:
Please provide more details about what you are testing with in tree upgrades here and in the commit message.